integration_tests/e2e/refer.cy.ts

```diff
@@ -38,4 +45,44 @@ context('Refer', () => {
     referralStartPage.shouldHaveProcessInformation()
     referralStartPage.shouldContainStartButtonLink()
   })
+
+  it('Shows the "confirm person" page when starting a new referral', () => {
+    cy.signIn()
+
+    const course = courseFactory.build()
+    const courseOffering = courseOfferingFactory.build()
+    const prisoner = prisonerFactory.build({
+      firstName: 'Del',
+      lastName: 'Hatton',
+      dateOfBirth: '1980-01-01',
+    })
+
+    const person = personFactory.build({
+      name: 'Del Hatton',
+      prisonNumber: prisoner.prisonerNumber,
+      currentPrison: prisoner.prisonName,
+      dateOfBirth: '1 January 1980',
+      ethnicity: prisoner.ethnicity,
+      gender: prisoner.gender,
+      religionOrBelief: prisoner.religion,
+      sentenceType: prisoner.imprisonmentStatusDescription,
+      setting: 'Custody',
+    })
+
+    cy.task('stubPrisoner', prisoner)
+
+    const path = referPaths.people.show({
+      courseId: course.id,
+      courseOfferingId: courseOffering.id,
+      prisonNumber: prisoner.prisonerNumber,
```

Comment:
Doesn't really make a difference, but given this is prison number and in our service, should we use `person.prisonNumber` here?